mod_deploy/controllers.py

```diff
@@ -61,7 +61,7 @@ def decorated_function(*args, **kwargs):
         return decorated_function
     return decorator
 
-
+# Check github deploy signature key (valid or not)
```

Comment:
This is not the correct way to document methods. You need to use docstrings (with """) inside the def's instead.